src/metrics.rs

```diff
@@ -76,7 +76,7 @@ pub struct PubSubMetrics {
     pub pubsub_slot: IntGaugeVec,
     pub pubsub_program_slot: IntGaugeVec,
     pub pubsub_account_slot: IntGaugeVec,
-    pub websocket_reconnects: IntCounterVec,
+    pub ws_connect_attempt: IntCounterVec,
```

Comment:
Counters should be in plural form (attempts)

Reply:
I thinks we can close this PR, as it seems to be logical to make use of `websocket_disconnects` metrics for the same purposes